Framework/Algorithms/inc/MantidAlgorithms/Q1DWeighted.h

```diff
@@ -76,18 +76,35 @@ class MANTID_ALGORITHMS_DLL Q1DWeighted : public API::Algorithm {
   void calculate(const API::MatrixWorkspace_const_sptr &);
   void finalize(const API::MatrixWorkspace_const_sptr &);
 
+  struct wedgeParameters {
```

Comment:
Why not to call it just Wedge as it is what it says. Also one could add the wedge comparison operator here in the struct that can be used when finding the mirror. There is a mixture wedge/sector in the code. We chose Sector in instrument viewer as it is a better name. But here, for historic reasons it was called wedges. I'd suggest in the algorithm we rename all to wedges.